votor/src/event_handler.rs

```diff
@@ -440,6 +440,20 @@ impl EventHandler {
     /// we never saw the parent ready for the first slot and haven't voted for first slot
     /// so we can't keep processing rest of the window. This is especially a problem for
     /// cluster standstill.
+    /// For example:
+    ///    A 40%
+    ///    B 40%
+    ///    C 30%
+    /// A and B finalize block together up to slot 9, now A exited and C joined.
+    /// C sees block 9 as finalized, but it never had parent ready triggered for slot 8.
+    /// C can't vote for any slot in the window because there is no parent ready for slot 8.
+    /// While B is stuck because it is waiting for >60% of the votes to finalize slot 9.
+    /// The cluster will get stuck.
+    /// After we add the following function, C will see that block 9 is finalized yet
+    /// it never had parent ready for slot 8, so it will find the block id of 8 from
```

Comment:
What if `C` started from a snapshot of `9` and has thus rooted `9`, and doesn't have `8` in its blockstore?